docs/programming-oak.md

```diff
@@ -368,20 +368,47 @@ using the following script:
 ./scripts/push_example -e hello_world
 ```
 
+### Creating a Permissions File
+
+A permissions file is a `.toml` file provided by the host owner, as an
+additional layer of defense. This file specifies the features of Oak that are
+permitted for the applications running on the host. In particular, using this
+file, the host owner can enable or disable gRPC or HTTP connections. In
+addition, it is possible to specify an allowlist of external gRPC or HTTP
+services that the applications can connect to. Interaction with all other
+external servers will be prohibited. Connections to insecure HTTP servers will
+be allowed regardless of the allowlist! However, Oak's IFC guarantees that only
+`public` data will be sent over to insecure external services.
```

Comment:
Also, can Oak applications request their permission list?
But I'm not sure if it's really necessary. Probably it would make sense, if applications are supposed to work with different permission combinations, and not fail.